cores/esp32/esp32-hal-gpio.c

```diff
@@ -95,12 +95,16 @@ extern void ARDUINO_ISR_ATTR __pinMode(uint8_t pin, uint8_t mode)
         log_e("Invalid pin selected");
 		return;
 	}
+
+    gpio_hal_context_t gpiohal;
+    gpiohal.dev = GPIO_LL_GET_HW(GPIO_PORT_0);
```

Comment:
Indentation issue here :)

Reply:
Wierd, I dont see that in GitHub Deskto / VS Code or GitHub integrated editor.

Reply:
REplaced tabs with spaces, should be fine now :)